data/icons/scalable/apps/printer-symbolic.svg

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is this icon the same as the `printer-symbolic.svg` included in the GNOME icon theme? If so, we don't need to include it here.